projects/packages/stats-admin/src/class-rest-controller.php

```diff
@@ -291,51 +355,6 @@ protected function get_has_never_published_post( $req ) {
 		return rest_ensure_response( $has_never_published_post );
 	}
 
-	/**
-	 * Query the WordPress.com REST API using the blog token
-	 *
-	 * @param String $path The API endpoint relative path.
-	 * @param String $version The API version.
-	 * @param array  $args Request arguments.
-	 * @param String $body Request body.
-	 * @param String $base_api_path (optional) the API base path override, defaults to 'rest'.
-	 * @param bool   $use_cache (optional) default to true.
-	 * @return array|WP_Error $response Data.
-	 */
-	protected function request_as_blog_cached( $path, $version = '1.1', $args = array(), $body = null, $base_api_path = 'rest', $use_cache = true ) {
```

Comment:
I'm happy with the fact that we don't have to do this in the rest controller any longer 😃